tool/lib/commands/generate_changelog.dart

```diff
@@ -16,6 +16,17 @@ class GenerateChangelogCommand extends Command {
   // You can authorize your access if you run into a github rate limit.
   // Don't check in your passwords or auth tokens.
   static const auth = '';
+
+  /// Commit names to ignore in changelog generation.
+  ///
+  /// We will only check for exact matches on this, after trimming out the
+  /// trailing commit number.
+  static const ignoredCommitNames = [
+    'update goldens',
+    'Update Goldens',
+    'Update goldens'
+  ];
```

Comment:
instead of storing all variations, why not just store a single lowercase string in the list 'update goldens', and then check `ignoredCommitNames.contains(message.toLowercase())`.

I can see this list getting large if we start to have more commit names we want to filter out. This also feels more robust. for example, if the commit name has a capitalization typo 'UPdate goldens' or 'update Goldens'.

Reply:
I figure that initially, it's better to risk not skipping something we want to skip than to skip something we don't want to skip.

Depending on how this gets used, feel free to broaden the scope of the filtering.